Externals/Logitech/Include/LogitechLEDLib.h

```diff
@@ -0,0 +1,171 @@
+//
+// Logitech Gaming LED SDK
+//
+// Copyright (C) 2011-2014 Logitech. All rights reserved.
```

Comment:
Free to use, I specifically mailed their support and that guy replied "Please feel free to include our header file and the static library in your project. There's already plenty of publicly available repositories that share our SDK."
Nothing more specific though.